index.js

```diff
@@ -57,7 +57,11 @@ function defaultConsumer (field, file, filename, encoding, mimetype, body) {
   file.on('data', data => { fileData.push(data) })
   file.on('limit', () => { lastFile.limit = true })
   file.on('end', () => {
-    lastFile.data = Buffer.concat(fileData)
+    if (!lastFile.limit) {
+      lastFile.data = Buffer.concat(fileData)
+    } else {
+      delete lastFile.data
```

Comment:
This should be a little bit faster
```suggestion
      lastFile.data = undefined
```